tests/integration_tests/security/test_jail.py

```diff
@@ -152,14 +254,21 @@ def test_cgroups(test_microvm_with_initrd):
     sys_cgroup = '/sys/fs/cgroup'
     assert os.path.isdir(sys_cgroup)
 
-    check_cgroups(cgroups, sys_cgroup, test_microvm.jailer.jailer_id)
+    if cgroup_version == 1:
+        check_cgroups_v1(cgroups, sys_cgroup, test_microvm.jailer.jailer_id)
+    else:
+        check_cgroups_v2(cgroups, sys_cgroup, test_microvm.jailer.jailer_id)
 
 
-def test_node_cgroups(test_microvm_with_initrd):
+def test_node_cgroups(test_microvm_with_initrd, sys_setup_cgroups):
     """Test the cgroups are correctly set by the jailer."""
+    # pylint: disable=redefined-outer-name
     test_microvm = test_microvm_with_initrd
     test_microvm.jailer.cgroups = None
     test_microvm.jailer.numa_node = 0
+    cgroup_version = sys_setup_cgroups
+    if cgroup_version == 2:
```

Comment:
Does anything break if we just `test_microvm.jailer.cgroup_ver = sys_setup_cgroups`?

Reply:
Nope, makes sense to assign it directly for sure.